javascript/javascript1/week3/NOnoN0nOYes (Note taking app).js

```diff
@@ -0,0 +1,56 @@
+const notes = [];
+
+function saveNote(content, id) {
+ return notes.push({ content, id });
+
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Eat my mommy", 3);
+saveNote("Eat zebra", 4);
+saveNote("Check my homework", 5);
+saveNote("Go to bathroom", 6);
+saveNote("Keep calm and drink caffe", 7);
+saveNote("My mom is stupid from Aria", 8);
+saveNote("Cook my dog that doesn't exist", 9);
+saveNote("Play on my moms belly", 10);
+
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+
+function getNote(id) {
+ if (typeof id !== "number" )
+ return console.log("Error");
+
+else {
+
+ for ( let i = 0; i <= notes.length; i++){
+  if (id == notes[i].id){
+return console.log(notes[i]);
+   }
+  }
+ }
+}
+
+getNote(5);
```

Comment:
what will be output for getNote(12) ? I think, try to use condtion for that if get id  doesn't exist than appeare message "Enter valid Id Number"

Reply:
ok